builder/yandex/template_func.go

```diff
@@ -32,5 +34,5 @@ func templateCleanImageName(s string) string {
 }
 
 var TemplateFuncs = template.FuncMap{
-	"clean_image_name": templateCleanImageName,
+	"clean_resource_name": templateCleanResourceName,
```

Comment:
Ah, I just found this one; should we add it too ? I think yes and I pushed a commit :D 

I'll leave the merging choice to you.

Reply:
( I think it might have been a copy paste )

Reply:
Yeah, we can do that. I considered it originally and decided not to since it has never received the deprecation warning, but it's going out in 1.5 so it can be backwards breaking.